compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -211,6 +211,8 @@ object Completion:
     val completer = new Completer(mode, pos, untpdPath, matches0)
 
     val result = adjustedPath match
+      // Ignore `package foo@@` and `package foo.bar@@`
+      case ((_: tpd.Select) | (_: tpd.Ident)):: (_ : tpd.PackageDef) :: _       => Map.empty
```

Comment:
The check is correct, but it may be better to make it in `Completion.completionMode` instead of this place. The reason is that despite filtering compiler completions, completion mode is also used for the rest of the completions in presentation compiler.